src/plugins/vis_type_table/README.md

```diff
@@ -1 +1,35 @@
-Contains the data table visualization, that allows presenting data in a simple table format.
\ No newline at end of file
+# Data Table
+
+This is an OpenSearch Dashboards plugin that is used to visualize data and aggregations in tabular format.
+
+## Create Data Table
+To create a data table in OpenSearch Dashboards, first select `Visualize` from the navigation menu. Then click `Create Visualization` and choose `Data Table` as the visualization type.
+
+## Select Metrics
+
+### Metrics Aggregation
+At the `Metrics`, select the metric aggregation type from the menu and config it accordinly.
```

Comment:
Do we want to clarify that by default, as you add more metrics, they will be added as new columns?

Reply:
Yeah I could add some clarifications.

```
At the `Metrics`, select the metric aggregation type from the menu and configure it accordingly. You could also add multiple metrics and each metrics is a separate column in table visualization.
```